common/persistence/dataInterfaces.go

```diff
@@ -2422,3 +2439,22 @@ func SplitHistoryGarbageCleanupInfo(info string) (domainID, workflowID, runID st
 	workflowID = info[len(domainID)+1 : workflowEnd]
 	return
 }
+
+// NewGetReplicationTasksFromDLQRequest creates a new GetReplicationTasksFromDLQRequest
+func NewGetReplicationTasksFromDLQRequest(
+	sourceClusterName string,
```

Comment:
why do we need a source cluster?

Reply:
DLQ has a source cluster index in it. So get operations will be per source cluster.